pyDeltaRCM/default.yml

```diff
@@ -0,0 +1,138 @@
+site_prefix:
+  type: 'str'
+  default: ''
+case_prefix:
+  type: 'str'
+  default: ''
+out_dir:
+  type: 'str'
+  default: 'deltaRCM_Output'
+Length:
+  type: 'float'
+  default: 1000.
+Width:
+  type: 'float'
+  default: 2000.
```

Comment:
I know these defaults are the same as the old .yaml file, but we should consider a smaller domain/case for the 'default' implementation. A simulation of this size doesn't exactly resolve quickly -- just something for us to keep in mind going forward.

Reply:
That's a good point. I just copied it parameter-for parameter. The parameters supplied in `tests` is really small though. What do you think is a good default?

Reply:
Yeah the test case is tiny just to make sure the parcel routing happens as expected. We probably will need to play around with different parameter sets once we're done with the code to get something that is quick and representative of what the model can do.

Reply:
Groovy. Save that for later then.